person.rb

```diff
@@ -3,7 +3,7 @@
 require_relative 'trimmer'
 
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [ ] A person must have many rentals. Kindly. Le's add a method for adding rentals  

```ruby
  def add_rental(...)
    ...
  end
```

- Note: The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association